EpsilonCCS/Src/main.c

```diff
@@ -208,10 +227,14 @@ void SystemClock_Config(void)
     RCC_OscInitTypeDef RCC_OscInitStruct;
     RCC_ClkInitTypeDef RCC_ClkInitStruct;
 
+    /**Configure the main internal regulator output voltage
```

Comment:
I don't think these comments will live through code generation. Unless this was generated, then ignore it.

Reply:
Was generated